stdlib/influxdata/influxdb/sample/list_test.flux

```diff
@@ -0,0 +1,27 @@
+package sample_test
+
+
+import "influxdata/influxdb/sample"
+import "csv"
+import "testing"
+
+expected = "
+#group,false,false,false
+#datatype,string,long,boolean
+#default,_result,,
+,result,table,_value
+,,0,true
+"
+
+t_list = () => {
```

Comment:
nit, we have stopped creating functions just for the test case. The new best practice is to simply put the logic directly into the testcase block.

```
testcase list {
    got = sample.list()
        |> count()
        |> map(fn: (r) => ({_value: r._value > 0}))
    want = csv.from(csv: expected)

    testing.diff(got: got, want: want)
}
```

Could you update this?